notebooks/finetuning.ipynb

```diff
@@ -0,0 +1,137 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "index = client.indexes.create_index(\n",
+    "    index_type=\"text\", # Specify the type of index as 'text'\n",
+    "    fields={\n",
+    "      # Defines fields that will be indexed, their values will influence relevance\n",
+    "      \"searchable\": [\"prod_name\", \"department_name\", \"detail_desc\", \"garment_group_name\", \"index_group_name\", \"index_name\", \"perceived_colour_master_name\", \"perceived_colour_value_name\"],\n",
+    "      # Defines fields that will be filterable\n",
+    "      \"filterable\": [\"department_name\", \"colour_group_name\", \"garment_group_name\", \"index_group_name\", \"index_name\"]\n",
+    "    }\n",
+    ")\n",
+    "index.status(watch=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Finetuned Index\n",
+    "\n",
+    "Finetuning requires feedback data. Namely, examples of queries with relevance judgements on their results. The data takes the form of `(query, object_id, label)`, where `object_id` is the ID of the search result in the Object Store, and `label` is the rating on a `{BAD, OK, GREAT}` scale. \n",
+    "\n",
+    "Here are example grades of a search for dark blue polo shirt:\n",
+    "\n",
+    "```json\n",
+    "{\"object_id\": \"570004012\", \"query\": \"dark blue polo shirt\", \"label\": \"GREAT\"}\n",
+    "{\"object_id\": \"806794003\", \"query\": \"dark blue polo shirt\", \"label\": \"GREAT\"}\n",
+    "{\"object_id\": \"751334002\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"570004069\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"570004078\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"689107004\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"806794004\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "{\"object_id\": \"689113002\", \"query\": \"dark blue polo shirt\", \"label\": \"GREAT\"}\n",
+    "{\"object_id\": \"570004106\", \"query\": \"dark blue polo shirt\", \"label\": \"BAD\"}\n",
+    "```\n",
+    "\n",
+    "[Complete example](https://d11p8vtjlacpl4.cloudfront.net/demo-data/graded_input.jsonl)\n",
+    "\n",
+    "These training labels are scoring results for a search on `dark blue polo shirt` as either `GREAT` or `BAD`. The Funetuning process will use this feedback to create a new Index optimized on the feedback. To download the full set of training labels and build a finetuned index with it:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = requests.get(\"https://d11p8vtjlacpl4.cloudfront.net/demo-data/graded_input.jsonl\")\n",
```

Comment:
Is this still where we should get the labeled data from given you are including the "graded_input.jsonl" file in the repository?

Reply:
Updated to use local file instead